trendyqc/trend_monitoring/models/bam_qc.py

```
@@ -406,3 +409,127 @@ class Duplication_metrics(models.Model):
+    median_cv_coverage = models.FloatField()
+    median_5prime_bias = models.FloatField()
+    median_3prime_bias = models.FloatField()
+    median_5prime_to_3prime_bias = models.FloatField()
+    pf_not_aligned_bases = models.FloatField()
+
+    class Meta:
+        app_label = "trend_monitoring"
+        db_table = "rnaseq_metrics"
+
+
+class RNA_seqc(models.Model):
```

Ruff (0.8.0)

Model does not define `__str__` method

(DJ008)